samples/sensor/mcux_acmp/README.rst

```diff
@@ -65,3 +66,14 @@ ACMP input voltage by changing the voltage input to J2-3.
    :board: frdm_ke17z512
    :goals: flash
    :compact:
+
+Building and Running for MIMXRT1180-EVK
+=======================================
+Build the application for the MIMXRT1180-EVK board, and adjust the
+ACMP input voltage by changing the voltage input to J45-13.
+
+.. zephyr-app-commands::
```

Comment:
This format won't work (see the rendered documentation here: https://builds.zephyrproject.io/zephyr/pr/77836/docs/samples/sensor/mcux_acmp/README.html#building-and-running-for-mimxrt1180-evk).

We need to provide two `zephyr-app-commands` entries, one per SOC target.